pygtfs/gtfs_entities.py

```diff
@@ -27,6 +26,7 @@ def _validate_date(*field_names):
     @validates(*field_names)
     def make_date(self, key, value):
         return datetime.datetime.strptime(value, '%Y%m%d').date()
+
```

Comment:
Again, intellij. But I think it makes sense in this case. There's this python style guide that says that you need 2 blank lines surrounding a top level function https://www.python.org/dev/peps/pep-0008/#blank-lines. But I'm happy to revert if you want to.